ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -0,0 +1,73 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "s3_log_system.h"
+
+#include <cstdarg>
+
+using namespace Aws::Utils;
+using namespace Aws::Utils::Logging;
+
+s3_log_system::s3_log_system(WT_EXTENSION_API *wtApi, uint32_t awsVerbose)
+{
+    static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping = {
+      {-3, Aws::Utils::Logging::LogLevel::Error}, {-2, Aws::Utils::Logging::LogLevel::Warn},
+      {-1, Aws::Utils::Logging::LogLevel::Info}, {0, Aws::Utils::Logging::LogLevel::Info},
+      {1, Aws::Utils::Logging::LogLevel::Debug}};
+    if (verbosityMapping.find(awsVerbose) != verbosityMapping.end()) {
+        logLevel = verbosityMapping.at(awsVerbose);
+    } else {
+        logLevel = Aws::Utils::Logging::LogLevel::Error;
```

Comment:
Namespace is not required here since it's declared above

Reply:
I'd prefer to remove the `using` statements. It makes it clear what bits of code are from the AWS SDK, and there isn't so much code that readability is affected.